netbox/ipam/tables/vlans.py

```diff
@@ -244,3 +247,75 @@ class Meta(NetBoxTable.Meta):
     def __init__(self, interface, *args, **kwargs):
         self.interface = interface
         super().__init__(*args, **kwargs)
+
+
+#
+# VLAN Translation
+#
+
+class VLANTranslationPolicyTable(NetBoxTable):
+    name = tables.Column(
+        verbose_name=_('Name'),
+        linkify=True
+    )
+    description = tables.Column(
+        verbose_name=_('Description'),
+        # linkify=True
+    )
+    tags = columns.TagColumn(
+        url_name='ipam:vlantranslationpolicy_list'
+    )
+
+    class Meta(NetBoxTable.Meta):
+        model = VLANTranslationPolicy
+        fields = (
+            'pk', 'id', 'name', 'description', 'tags', 'created', 'last_updated',
+        )
+        default_columns = ('pk', 'name', 'description')
+
+
+class VLANTranslationRuleTable(NetBoxTable):
+    policy = tables.Column(
+        verbose_name=_('Policy'),
+        linkify=True
+    )
+    local_vid = tables.Column(
+        verbose_name=_('Local VID'),
+        linkify=True
```

Comment:
IMO we should generate an index number for each rule. Within a policy, rules should be ordered by `local_vid`, but the table should include a sequential line number indicating the rule's position in the policy. This could be used as the link to view an individual rule.

Reply:
I feel like this is more effort than it's worth. I've made the `id` column visible by default, which ensures there's a distinct clickable value column where the value will be persistent regardless of how the user sorts or filters the table.